index.js

```diff
@@ -399,16 +399,6 @@ XiaomiRoborockVacuum.prototype = {
 
                 // STATE
                 log.debug('DEB getDevice | Initializing status values to variables');
-                that.cleaning = state.cleaning;
-                that.charging = state.charging;
-                that.docked = state.charging;
-                that.speed = state.fanSpeed;
-                that.battery = state.batteryLevel;
-                if(state.cleaning == true) {
-                    that.pausepossible = true;
-                } else {
-                    that.pausepossible = false;
-                }
```

Comment:
Sure? If the vacuum cleaner robot cleans and Homebridge is restarted, HomeKit will not show that it cleans. Or? Can you test that?

Reply:
If we move the indicators for the get methods to actually hit the API, we shouldn't need to store any state variable. Should we? But I'll come back with a definitive answer after testing the scenario you suggest :)